reason_4.0/lib/core/minisite_templates/modules/events.php

```diff
@@ -4312,152 +4229,12 @@ function get_registration_forms_markup($event)
 	}
 
 	/**
-	 * Redirect to empty date if the given requested date is not a date the event occurrs
```

Comment:
A bit puzzled about the removal of this method -- it seems like a general-purpose method unrelated to slot registration